diff --git a/node_modules/storybook/dist/_node-chunks/builder-manager-EFOAQ5X5.js b/node_modules/storybook/dist/_node-chunks/builder-manager-EFOAQ5X5.js
index 0000000..0000000 100644
--- a/node_modules/storybook/dist/_node-chunks/builder-manager-EFOAQ5X5.js
+++ b/node_modules/storybook/dist/_node-chunks/builder-manager-EFOAQ5X5.js
@@ -622,6 +622,8 @@
     "version"
   ],
   "react-dom/client": ["createRoot", "hydrateRoot"],
+  "react/jsx-runtime": ["jsx", "jsxs", "Fragment"],
+  "react/jsx-dev-runtime": ["jsx", "jsxs", "jsxDEV", "Fragment"],
   "@storybook/icons": [
     "AccessibilityAltIcon",
     "AccessibilityIcon",
@@ -1253,6 +1255,8 @@
   react: "__REACT__",
   "react-dom": "__REACT_DOM__",
   "react-dom/client": "__REACT_DOM_CLIENT__",
+  "react/jsx-runtime": "__REACT_JSX_RUNTIME__",
+  "react/jsx-dev-runtime": "__REACT_JSX_RUNTIME__",
   "@storybook/icons": "__STORYBOOK_ICONS__",
   "storybook/manager-api": "__STORYBOOK_API__",
   "storybook/test": "__STORYBOOK_TEST__",
diff --git a/node_modules/storybook/dist/manager/globals-runtime.js b/node_modules/storybook/dist/manager/globals-runtime.js
index 0000000..0000000 100644
--- a/node_modules/storybook/dist/manager/globals-runtime.js
+++ b/node_modules/storybook/dist/manager/globals-runtime.js
@@ -29668,6 +29668,8 @@
   react: "__REACT__",
   "react-dom": "__REACT_DOM__",
   "react-dom/client": "__REACT_DOM_CLIENT__",
+  "react/jsx-runtime": "__REACT_JSX_RUNTIME__",
+  "react/jsx-dev-runtime": "__REACT_JSX_RUNTIME__",
   "@storybook/icons": "__STORYBOOK_ICONS__",
   "storybook/manager-api": "__STORYBOOK_API__",
   "storybook/test": "__STORYBOOK_TEST__",
@@ -76650,6 +76652,8 @@
   react: REACT,
   "react-dom": REACT_DOM,
   "react-dom/client": REACT_DOM_CLIENT,
+  "react/jsx-runtime": __toESM(require_jsx_runtime(), 1),
+  "react/jsx-dev-runtime": __toESM(require_jsx_runtime(), 1),
   "@storybook/icons": dist_exports2,
   "storybook/manager-api": manager_api_exports,
   "storybook/theming": theming_exports,
